include/fiction/utils/truth_table_utils.hpp

```diff
@@ -127,6 +127,63 @@ inline kitty::dynamic_truth_table create_xnor_tt() noexcept
 
     return table;
 }
+/**
+ * Creates and returns a truth table that implements an identity function in two variables.
+ *
+ * @return identity function in two variables.
+ */
+inline kitty::dynamic_truth_table create_double_wire_tt() noexcept
+{
+    const char* hex_string = "11100100";
```

Comment:
All the `hex_string` variables are defined as binary strings. Maybe they can be renamed accordingly?

Also, they can probably all be `static constexpr`.